autoupdater/docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+name: "ap_autoupdater"
```

Comment:
I suggest we keep the original name ‘watchtower’ as it is already familiar to people. Changing the name might add unnecessary confusion to its implementation.